packages/gestalt/src/Contents.js

```diff
@@ -483,7 +483,7 @@ export default class Contents extends React.Component<Props, State> {
     const visibility = mainDir === null ? 'hidden' : 'visible';
     const background = `${bgColor}Bg`;
     const stroke = bgColor === 'white' ? '#efefef' : null;
-    const borderColor = bgColor === 'white' ? 'lightGray' : bgColor;
+    const borderColor = bgColor;
```

Comment:
You can probably clean this up more by getting rid of the variable all together :)

Reply:
Removed the variable altogether. :)